### PR TITLE
Add a run task for TPCH queries

### PR DESCRIPTION
**Pull request by onderkalaci**
Steps to install & run various TPCH suites

```
fab set_pg_latest:/home/ec2-user/pg-950-citus-600
fab use.postgres:9.5.0 use.citus:v6.0.1 setup.basic_testing
fab add.tpch:scale_factor=1
fab pg.stop


fab set_pg_latest:/home/ec2-user/pg-960-citus-600
fab use.postgres:9.6.1 use.citus:v6.0.1 setup.basic_testing
fab add.tpch:scale_factor=1
fab pg.stop

fab set_pg_latest:/home/ec2-user/pg-950-citus-610
fab use.postgres:9.5.0 use.citus:v6.1.0-rc1 setup.basic_testing
fab add.tpch:scale_factor=1
fab pg.stop


fab set_pg_latest:/home/ec2-user/pg-960-citus-610
fab use.postgres:9.6.1 use.citus:v6.1.0-rc1 setup.basic_testing
fab add.tpch:scale_factor=1
fab pg.stop


fab pg.stop
fab set_pg_latest:/home/ec2-user/pg-950-citus-600
fab pg.start
fab run.tpch_queries 

fab pg.stop
fab set_pg_latest:/home/ec2-user/pg-960-citus-600
fab pg.start
fab run.tpch_queries 



fab pg.stop
fab set_pg_latest:/home/ec2-user/pg-950-citus-601
fab pg.start
fab run.tpch_queries 

fab pg.stop
fab set_pg_latest:/home/ec2-user/pg-960-citus-601
fab pg.start
fab run.tpch_queries 
```

### CODE REVIEW

File: fabfile/utils.py [hunk 1]
```diff
@@ -22,6 +22,10 @@ def psql(command):
     with cd(config.paths['pg-latest']):
         return run('bin/psql -c "{}"'.format(command))
 
+def psql_file(path):
+    with cd(config.paths['pg-latest']):
+        return run('bin/psql -f ' + path + ' | grep Time:')
```
Comment (onderkalaci) [inline fabfile/utils.py]: Could we get time in a more intuitive way? I.e., could we get time from `run` command?

**Comment by lithp** [comment]
Is this blocking you? Planning on looking at it after testing

**Comment by onderkalaci** [comment]
> Is this blocking you? Planning on looking at it after testing

Nope, I'm using this branch to test TPCH but we can merge it later on

**Comment by metdos** [comment]
@onderkalaci, most likely you know this, just wanted to note. If you need these stuff for testing, you can simply switch to this branch in your test cluster.

**Comment by metdos** [comment]
> @onderkalaci, most likely you know this, just wanted to note. If you need these stuff for testing, you can simply switch to this branch in your test cluster.

Ohh, I was late. Deleting these comments :)

**Comment by onderkalaci** [comment]
I'm also thinking that with spending 1-2 days we could add #56 to this PR as well.

**Pull request closed without merging**